src/client/Microsoft.Identity.Client/Internal/Requests/SilentRequest.cs

```diff
@@ -152,6 +152,7 @@ private async Task<MsalTokenResponse> ExecuteBrokerAsync(CancellationToken cance
         private async Task<AuthenticationResult> CreateAuthenticationResultAsync(MsalAccessTokenCacheItem cachedAccessTokenItem)
         {
             var msalIdTokenItem = await CacheManager.GetIdTokenCacheItemAsync(cachedAccessTokenItem.GetIdTokenItemKey()).ConfigureAwait(false);
+            AuthenticationRequestParameters.RequestContext.ServiceBundle.TelemetryManager.SuccessfulSilentCallCount++;
```

Comment:
I don't think this needs to tracked separately. Doesn't "event in progress" already know about the number of silent calls made?

Reply:
i couldn't find where this is tracked in the api events. lmk if you do.

---
In reply to: [389772910](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1693#discussion_r389772910) [](ancestors = 389772910)